src/schema/v2/me/watchedLotConnection.ts

```diff
@@ -31,6 +31,12 @@ export const WatchedLotConnection: GraphQLFieldConfig<void, ResolverContext> = {
 
     const { body, headers } = await saleArtworksAllLoader(gravityArgs)
 
+    // TODO: Move this property to gravity JSON response
+    body.forEach((saleArtwork) => {
+      saleArtwork.isWatching = true
+      return saleArtwork
```

Comment:
Do we have any other examples of introducing an `isWatching` property that doesn't already exist on json type? What I mean here is that it's one thing to take a field like a time or `sellingPriceCents` and add a resolver to get formatted strings, etc - In this case we are just adding a new property that is very tied to the context of the request. Likewise for moving it into gravity - `SaleArtwork` has no knowledge of users so that json may not belong there. Sorry I feel like i could have typed this all in one sentence!

Reply:
We do do this occasionally. I've updated the field description with a note about where context lies and added a todo. I think all of this will resolve automatically once we migrate bidding engine.